operator/v1/types_network.go

```diff
@@ -507,6 +507,26 @@ type GatewayConfig struct {
 	// The supported values are "Restricted" and "Global".
 	// +optional
 	IPForwarding IPForwardingMode `json:"ipForwarding,omitempty"`
+	// V4InternalMasqueradeSubnet contains the v4 masquerade addresses used internally by
+	// ovn-kubernetes to enable host to service traffic. The host is configured with these
+	// addresses, as well as the shared gateway bridge interface. The values can be changed after
+	// installation. The subnet chosen should not overlap with other networks specified for
+	// OVN-Kubernetes as well as other networks used on the host. Additionally the subnet must
+	// be large enough to accommodate 6 IPs (maximum prefix length /29).
+	// The default subnet is 169.254.169.0/29
```

Comment:
I think to fit in with standard prose
```suggestion
	// When omitted, this means no opinion and the platform is left to choose a reasonable default which is subject to change over time. Currently the default subnet is 169.254.169.0/29.
```